docs/howto/how-to-make-tensorflow-plugin-with-ui/tensorflow-plugin-with-ui.md

```diff
@@ -0,0 +1,1488 @@
+      <Grid item xs>
+        <TextField
+          fullWidth
+          type="number"
+          id="sequence-length"
+          label="Sequence Length"
+          onChange={(e) => onHandleTextChange(e, 'textVectorizationConfig', 'sequenceLength')}
+          value={model.Configuration.textVectorizationConfig.sequenceLength || 250}
+        />
+
+      </Grid>
+    </Grid>
+    </Container>
+    <Container>
+          <Typography variant="h1" gutterBottom>
+      Model and Training Configuration
+    </Typography>
+    <Grid alignItems="flex-end" container spacing={2}>
+    <ModelSection />
+    </Grid>
+    </Container>
+
+    </Box>
+  );
+};
+
+
+const App = () => {
+  return (
+    <Box marginTop={3}>
+      <Container>
+        <Grid container spacing={3}>
+          <Grid item xs={12}>
+          
+          </Grid>
+          <Grid item xs={12}>
+            <Explorer />
+          </Grid>
+        </Grid>
+      </Container>
+    </Box>
+  )
+}
+
+const datasetConfig = {
+  dataUrl: "",
+  datasetTargetDir: ".",
+  batchSize: 32,
+  seed: 42,
+  shouldCache: true,
+  trainingSetDir: "aclImdb/train",
+  testSetDir: "aclImdb/test", 
+}
+
+const datasetInfo = {
+  rawTrainSample: 
+    {textBatch: [], labelBatch: []}
+  ,
+  vectorizedSample: {
+    sampleRaw: "placeholder sample",
+    sampleVectorized: "09 90 1 1 0 0 12",
+  },
+  vocabIntValue: 0,
+  vocabTranslation: '',
+}
+
+const textVectorizationConfig = {
+  sequenceLength: 250,
+  outputMode: 'int',
+  outputModeIndex: 0,
+  outputModeAnchorEl: null,
+  tokenChips: [],
+  translationRequest: {
+    token: "",
+  }
+}
+
+const modelConfig = {
+  embeddingDim: 16,
+  showSummary: false,
+  modelName: "text-classifier-model"
+}
+
+const modelEvaluation = {
+  loss: 0.0,
+  accuracy: 0.0,
+  history: {
+    trainingLoss: [],
+    trainingBinaryAccuracy: [],
+    validationLoss: [],
+    validationBinaryAccuracy: [],
+  }
+}
+
+const trainingConfig = {
+  epochs: 4,
+  maxFeatures: 10000,
+}
+
+const Configuration = {
+  plotUri: [],
+  datasetConfig,
+  textVectorizationConfig,
+  modelConfig,
+  trainingConfig,
+  modelEvaluation,
+  datasetInfo,
+}
+
+const defaultConfig = {
+  Configuration,
+}
+
+const Tool = () => {
+  return (
+    <DesignerApi messages={{}} defaultConfig={{...defaultConfig}}>
+      <AyxAppWrapper> 
+        <App />
+      </AyxAppWrapper>
+    </DesignerApi>
+  )
+}
+
+ReactDOM.render(
+  <Tool />,
+  document.getElementById('app')
+);
+```
+While it has an impressive line count by default, our only non-boilerplate additions are skeleton frames in our return statement and default configuration defined and detailed by the `alteryx-ui` API and docs.
+
+### Module: `./src/constants.tsx`
+
+```tsx
+export const DS_CONFIG = 'datasetConfig'
+export const MDL_CONFIG = 'modelConfig'
+export const EVAL_CONFIG = 'modelEvaluation'
+```
+
+Next, we need a directory to store our components. In addition, a subdirectory for our charting component.
+You should now have
```

Comment:
TW review:

Next, we need a directory to store the components. In addition, a subdirectory for the charting component.
You should now have...